Filipino/bagong simula/ipis/ipis.md

```diff
@@ -4,6 +4,6 @@ iyong kamay pra subukang takpan ang iyong mata. Sa pag angat mo ng kamay mo,
 napansin mong may kakaibang hugis na mistulang nakadikit sa iyong palad.
 Sa puntong iyon, nalaman mo na hindi lang ikaw na nasa loob ng kwarto.
 
-Subukang [durugin ang ipis](http://www.publicdomainpictures.net/pictures/60000/velka/cockroach.jpg).
+Subukang [durugin ang ipis](durog/durog.md).
```

Comment:
I've deleted the last path used by this story to reflect the next story which I've made and added the deleted link to my story